db/stats.go

```diff
@@ -29,28 +28,20 @@ func GetNftCount(conn *pgxpool.Conn, q QueryNftCountRequest) (count QueryCountRe
 }
 
 func GetNftTradeStats(conn *pgxpool.Conn, q QueryNftTradeStatsRequest) (res QueryNftTradeStatsResponse, err error) {
-	payload := struct {
-		Type    string `json:"@type"`
-		Grantee string `json:"grantee"`
-	}{
-		"/cosmos.authz.v1beta1.MsgExec",
-		q.ApiAddress,
-	}
-	payloadJSON, err := json.Marshal(&payload)
-	if err != nil {
-		err = fmt.Errorf("marshal nft trade stats payload failed: %w", err)
-		logger.L.Error(err, q)
-		return
-	}
-
 	sql := `
-	SELECT count(*), sum((tx #>> '{"tx", "body", "messages", 0, "msgs", 0, "amount", 0, "amount"}')::bigint)
-	from txs
-	WHERE tx #> '{"tx", "body", "messages", 0}' @> $1
+	SELECT COUNT(*), sum((tx #>> '{"tx", "body", "messages", 0, "msgs", 0, "amount", 0, "amount"}')::bigint)
```

Comment:
Can add comment explaining the meaning of this sum of message 0? Now we cannot know it is a authz grant message just by reading code